hazelcast/src/main/java/com/hazelcast/core/HazelcastInstance.java

```diff
@@ -328,6 +328,11 @@
 
     /**
      * Returns all {@link DistributedObject}'s such as; queue, map, set, list, topic, lock, multimap.
+     * <p>
+     * The results are returned on a best-effort basis. The result might miss
+     * just-created objects and contain just-deleted objects. An existing
+     * object can also be missing from the list occasionally. One cluster
+     * member is queried to obtain the list.
```

Comment:
Should "one cluster member is queried" be really part of the method's contract? I am not expecting this to change anytime soon, but it feels like an implementation detail.

Reply:
It's a performance hint. If you want to use this method frequently, you'd like to know this.